File: s2-ex8.py
#Ler dois números n1 e n2 e exibir todos os números entre n1 e n2 

n1 = int(input("Insira o primeiro número: "))
n2 = int(input("Insira o segundo número: "))

print(f"Números entre {n1} e {n2}:", end=" ")
for n in range(n1+1, n2):
    print(n, end=" ")

File: s2-ex8.c
/* Ler dois números n1 e n2
Exibir todos os números entre n1 e n2 */

#include <stdio.h>

int main(void){

    int n1, n2;

    printf("Insira dois números: ");
    scanf("%d %d", &n1, &n2);

    printf("Números entre %d e %d: ", n1, n2);
    for (int n = n1+1; n < n2; n++) 
    printf("%d ", n);

    return 0;
}